.github/workflows/semantic-pull-request.yml

```diff
@@ -0,0 +1,46 @@
+name: "Check PR title format"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # Configure which types are allowed (newline delimited).
+          # Default: https://github.com/commitizen/conventional-commit-types
+          # Our specific types/scopes (from https://github.com/dashpay/dash/blob/master/CONTRIBUTING.md#contributor-workflow)
+          types: |
+            feat
+            fix
+            docs
+            style
+            refactor
+            perf
+            test
+            build
+            ci
+            chore
+            revert
+            trivial
+            backport
+          # Configure which scopes are allowed (newline delimited).
+          scopes: |
```

Comment:
It's possible to eliminate the scopes section here. In that case it would check the types, but allow any scope. I don't think we want to do that, but it's an option :man_shrugging: